src/TopBar/index.tsx

```diff
@@ -0,0 +1,47 @@
+import * as classnames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+
+import Logo from "../Logo";
+import { FlexBox, Justify } from "../flex";
+import TopBarButton from "./TopBarButton";
+import TopBarMenu from "./TopBarMenu";
+
+import "./index.less";
+
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  logoHref: PropTypes.string.isRequired,
+  title: PropTypes.node,
+};
+
+/**
+ * Global page-level header component.
+ */
+export default class TopBar extends React.PureComponent {
```

Comment:
Can we make this a named export? The explicit nature of named exports make for an easier development experience IMO, and it's the pattern we've been moving towards elsewhere. (also applies to other components in this PR)

Reply:
I agree, was just trying to follow convention. I'll happily switch - Glad we're moving in that direction.